dts/bindings/sd/zephyr,mmc-disk.yaml

```diff
@@ -19,3 +19,9 @@ properties:
       - 1
       - 4
       - 8
+
+  disk-name:
```

Comment:
isn't this what the label property is for

Reply:
Yes. However, we currently have the "label" property marked as deprecated in Zephyr. Would you prefer I use the deprecated property over `disk-name`? The reason I chose this property was because it follows the pattern used by other disk drivers.

Reply:
>However, we currently have the "label" property marked as deprecated in Zephyr.

That is the reason why `disk-name` was introduced. At this point, it would be much more harmful to use a label property instead.

Reply:
> > However, we currently have the "label" property marked as deprecated in Zephyr.
> 
> That is the reason why `disk-name` was introduced. At this point, it would be much more harmful to use a label property instead.

don't agree with this, and don't know your reasoning for saying it would be harmful, I am going to undeprecate the label property

Reply:
still think this should be the label property but i was missing some info, see my dismiss message